packages/roosterjs-editor-plugins/lib/plugins/Paste/sanitizeLinks/sanitizeLinks.ts

```diff
@@ -0,0 +1,35 @@
+import { chainSanitizerCallback } from 'roosterjs-editor-dom';
+import { HtmlSanitizerOptions } from 'roosterjs-editor-types';
+
+const HTTP = 'http:';
+const HTTPS = 'https:';
+
+/**
+ * @internal
+ * Clear local paths and remove link
+ * @param sanitizingOption the sanitizingOption of BeforePasteEvent
+ * */
+export default function sanitizeLinks(sanitizingOption: Required<HtmlSanitizerOptions>) {
+    ['href'].forEach(property => {
+        chainSanitizerCallback(
+            sanitizingOption.attributeCallbacks,
+            property,
```

Comment:
I think we can just add 'href' property here since we are only sanitizing a single property

Reply:
Oh my god!  I copied a code I did in the past and didn't noticed that. Thanks.